Gemfile

```diff
@@ -0,0 +1,13 @@
+
+# Pull gem index from rubygems
+source 'https://rubygems.org'
+
+# Pin the version of bundle we support
+gem 'bundler', '2.2.29'
+
+group :test do
+  gem 'rspec-core', '~> 3.10.1'
+  gem 'rspec-rails', '~> 4.0.2'
```

Comment:
Do we need this gem or could RSpec core suffice?

Reply:
good calll, we don't need `rspec-rails`. The `rspec-collections-matchers` I leave though, as that gives us the `expect()` function, and the `rspec_junit_formatter` should allow jenkins to give us a pretty breakdown of our individual test outputs.